website/docs/hardware/bill-of-materials/index.md

```diff
@@ -1,31 +0,0 @@
----
-description: "Complete lists of all components needed to build OpenArm, including mechanical parts, electronics, and off-the-shelf components."
----
-
-# 🧩 Bill of Materials
-
-This checklist includes **all components** required for assembling the **OpenArm bimanual setup**.
-
----
-
-## ✅ Actuators
-
-- [x] **4×** DAMIAO **DM-J8009-2EC** (Cost: ¥205,790)
-- [x] **2×** DAMIAO **DM-J4340P-2EC** (Cost: ¥42,066)
-- [x] **2×** DAMIAO **DM-J4340-2EC** (Cost: ¥37,387)
-- [x] **8×** DAMIAO **DM-J4310-2EC V1.1** (Cost: ¥112,113)
-
----
-
-## 🛠️ Mechanical Components
-
-- [x] [All manufactured arm parts](arm-manufactured) (Cost: ¥306,026)
-- [x] [All off-the-shelf arm parts](arm-off-the-shelf) (Cost: ¥34,336)
-- [x] [All pedestal components](pedestal) (Cost: ¥20,222)
-- [x] [All gripper components](gripper) (Cost: ¥117,930)
-
----
-
-## ⚡ Electrical Components
-
-- [x] [All electrical components](electrical) (Cost: ¥93,544)
```

Comment:
@pruthakj It seems that ¥93,544 is wrong. Auto computed total cost is ￥84,904: https://docs.openarm.dev/hardware/bill-of-materials/electrical
Could you check the total cost of all electrical components?

Reply:
Thanks for checking this.

We don't need #151. Because this PR computes the total cost from existing data automatically. :-)